proc/threads.go

```diff
@@ -123,122 +123,97 @@ func (tbe ThreadBlockedError) Error() string {
 	return ""
 }
 
-// Set breakpoints for potential next lines.
-func (thread *Thread) setNextBreakpoints() (err error) {
-	if thread.blocked() {
-		return ThreadBlockedError{}
+// returns topmost frame of selected goroutine
+func (dbp *Process) topframe() (Stackframe, error) {
```

Comment:
I want to get away from adding methods to `Process`. Can you make this a function that takes the goroutine and thread, it seems that's all it really needs from `Process`.